wiki/People/Users_with_unique_titles/en.md

```diff
@@ -352,6 +357,7 @@ These players earned their titles through either a crazy gameplay achievement or
 | ![][flag_FR] [Supairo](https://osu.ppy.sh/users/2837231) | Elite Mapper: Aspirant |
 | ![][flag_US] [SWAGGYSWAGSTER](https://osu.ppy.sh/users/7813296) | osu! Champion |
```

Comment:
```suggestion

```
His title was removed after the whole tournament ordeal a couple of months back.

Reply:
yeah